x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/translations.ts

```diff
@@ -37,6 +39,16 @@ export const CONDITION_FIELD_TITLE: { [K in MacosLinuxConditionEntry['field']]:
     'xpack.securitySolution.trustedapps.logicalConditionBuilder.entry.field.path',
     { defaultMessage: 'Path' }
   ),
+  'process.code_signature': i18n.translate(
```

Comment:
where is this used?

Reply:
It's probably not possible to create this in the form, but it is in our types describing trusted app, so I had to add it to keep proper type safety constraints. It will probably never appear (unless you manually call API with such data).